content/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-groups.md

```diff
@@ -1,44 +1,47 @@
 ---
+linkTitle: Collision Groups
+title: PhysX Collision Groups
 description: ' Create collision groups for the PhysX system in Open 3D Engine. '
-title: Collision Groups
-weight: 300
+weight: 400
+toc: true
 ---
 
-Use collision groups to define the layers that a collider interacts with. A collision group is similar to a mask with specific bits set in which each bit corresponds to a collision layer.
+Collision groups act as masks for collision layers. You can specify which collision layers are part of a collision group. A collision layer can be included in multiple collision groups. Each PhysX collider component can interact with one collision group. The collider can interact with any collider assigned to a collision layer included in the selected collision group.
 
-Two colliders interact if their collision layers are in each other's collision group. If one collision layer is not present in the other layer's collision group, the colliders don't interact.
+Colliders interact if their collision layers are in each other's collision group. If one collision layer is not present in the other layer's collision group, the colliders don't interact.
```

Comment:
[suggestion] The sentence `Colliders interact if their collision layers are in each other's collision group.` is the key part to understand how groups work, maybe it could be highlighted in bold?